homework-20/hw20-task1.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+def load_data(file_path):
+    with open(file_path, 'r', encoding='utf-8') as file:
```

Comment:
გამომდინარე იქიდან რომ უკვე შესწავლილი გვაქ Error Handling-ი  რეკომენდაციას გავუწევდი კოდში მის გათვალისწინებასაც, მაგალითად json ფაილის დამუშავების ფუნქციაში: 
    try:
        with open(path, 'r') as file:
            return json.load(file)
    except FileNotFoundError:
        print(f"Error: File {path} not found.")
        return None